src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,30 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+interface Props {
+  todo: Todo;
+  setSelectedTodoId: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, setSelectedTodoId }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(todo?.userId)
```

Comment:
You are directly interacting with the DOM by calling the getUser function. To follow React best practices, you should not interact with the DOM directly. Instead, you should use React's state and lifecycle methods to handle data fetching. This is a code quality issue.